server/user-topics/user-topics.routes.js

```diff
@@ -0,0 +1,9 @@
+const UserTopicsRouter = require('express').Router();
+const UserTopicCtrl = require('./user-topics.controller');
+const TopicCtrl = require('../topics/topic.model');
+
+UserTopicsRouter.post('/addTopic', UserTopicCtrl.addUserTopic);
```

Comment:
dont need this route since UserTopics add function will be called in the middleware chain of topics router